app/src/main/res/layout/fragment_ai_proposition.xml

```diff
@@ -0,0 +1,89 @@
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:theme="@style/AiCursorAndPrimaryColorTheme">
+
+    <com.google.android.material.appbar.MaterialToolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:titleCentered="true"
+        app:titleTextAppearance="@style/H2"
+        tools:title="@string/aiPromptTitle" />
+
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/nestedScrollView"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1">
+
+        <TextView
+            android:id="@+id/propositionTextView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingHorizontal="@dimen/marginStandardMedium"
+            android:paddingVertical="@dimen/marginStandard"
+            android:textIsSelectable="true"
+            tools:text="Objet : Candidature au poste de Manager de restaurant - \n\nCher(e) [Nom du responsable du recrutement],\n\nJ'ai le plaisir de vous envoyer mon CV en tant que candidat(e) au poste de Manager de restaurant au sein de votre établissement. Je suis convaincu(e) que mon expérience et mes compétences seraient un atout précieux pour votre équipe.\n\nJ'ai acquis une solide expérience dans la gestion d'équipe, le service client et l'optimisation des opérations dans l'industrie de la restauration.\n\nJe serais ravi(e) de discuter en détail de ma candidature lors d'un entretien avec vous. Vous trouverez mon CV en pièce jointe.\n\nMerci de votre attention.\nCordialement,\n[Votre nom]" />
+
+    </androidx.core.widget.NestedScrollView>
+
+    <com.google.android.material.card.MaterialCardView
+        style="@style/CardViewMail"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom"
+        app:cardElevation="@dimen/newMessageBottomBarElevation">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:paddingHorizontal="@dimen/marginStandardMedium"
+            android:paddingVertical="@dimen/alternativeMargin">
+
+            <com.google.android.material.button.MaterialButton
+                style="@style/TextButtonSecondary"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/suisseintl_regular"
+                android:text="@string/aiButtonAdjust"
+                android:textColor="@color/secondaryTextColor"
+                app:icon="@drawable/ic_filter"
+                app:iconTint="@color/iconColorSecondaryText" />
```

Comment:
The icon is not start aligned with the other views because of the padding inside the button

Reply:
Aligning the ripple, the touch area and the actual text (not taking into account the background) is more important imo. If we try to align the the icon+text of the button to the left of the screen we'd end up with strangely distributed proportions across the screen